.golangci.yaml

```diff
@@ -119,23 +119,23 @@ linters:
     - gocyclo
     - gofmt
     - gofumpt
-    - golint
+    # - golint
     - gomnd
     - gomodguard
     - goprintffuncname
     - gosec
     - gosimple
     - govet
     - grouper
-    - ifshort
+    # - ifshort
```

Comment:
This doesn't have any replacement advertised by golangci-lint but is deprecated.

Reply:
Checked ifshort, does not seem to work in current code base as well, though a useful linter reminder to have for shortening if conditions. Might have to let this go.